### PR TITLE
CLI

### PR DESCRIPTION
**Pull request by bkeepers**
This is a CLI client for the webhook proxy server in #8. It starts up and connects to the webhook proxy, forwarding any requests to a local server.

```
$ bin/hooksy --help

  Usage: hooksy [options]


  Options:

    -u, --url <url>    URL of the webhook proxy service (default: http://github-webhook-proxy.herokuapp.com/)
    -p, --port <n>     Local HTTP server port (default: 3000)
    -P, --path <path>  URL path to post proxied requests to` (default: /)
    -h, --help         output usage information
```

Output of it proxying a request:

```
$ ./bin/hooksy -u http://github-webhook-proxy.herokuapp.com/LXtFenSO2LcnQK8 -P /foobar
Proxying requests from http://github-webhook-proxy.herokuapp.com/LXtFenSO2LcnQK8 to http://127.0.0.1:3000/foobar
Connected http://github-webhook-proxy.herokuapp.com/LXtFenSO2LcnQK8
POST http://127.0.0.1:3000/foobar - 200
```

Output if there is an error:

```
$ ./bin/hooksy -u http://github-webhook-proxy.herokuapp.com/LXtFenSO2LcnQK8 -P /foobar
Proxying requests from http://github-webhook-proxy.herokuapp.com/LXtFenSO2LcnQK8 to http://127.0.0.1:3000/foobar
Connected http://github-webhook-proxy.herokuapp.com/LXtFenSO2LcnQK8

{ Error: connect ECONNREFUSED 127.0.0.1:3000
    at Object._errnoException (util.js:1024:11)
    at _exceptionWithHostPort (util.js:1046:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1182:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 3000,
  response: undefined }
```

- [x] Pick a name (domain name and npm package, [see discussion](https://probot-talk.slack.com/archives/C6Z70SRGD/p1512653933000584))
- [ ] Make the output ✨ 
- [x] Tests

### CODE REVIEW

**Comment by bkeepers** [comment]
> Make the output ✨

I'm going to do this in a separate PR. I'd love to get this merged so we can start using it.

File: client/test/index.test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+const createServer = require('../..')
+const Client = require('..')
```
Comment (JasonEtco) [inline client/test/index.test.js]: Any reason you chose to name it `Client` here and `Proxy` in the real thing?

File: client/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "main": "index.js",
+  "bin": {
+    "smee": "./bin/smee.js"
+  },
+  "scripts": {
+    "test": "jest --coverage && standard"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/probot/webhooks.git"
```
Comment (JasonEtco) [inline client/package.json]: Should we changes this to point to `probot/smee` (and then change the repo itself of course)?

File: client/test/index.test.js (hunk 1 above)
Comment (bkeepers) [reply client/test/index.test.js]: Nope. Fixed by renaming to `Client` everywhere.

File: client/package.json (hunk 2 above)
Comment (bkeepers) [reply client/package.json]: 🔜

**Pull request merged**